test-packages/memory-tests/sdk-v1/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@sap-cloud-sdk/memory-tests-v1",
+  "version": "0.0.1",
+  "description": "Memory usage tests of the SAP Cloud SDK for JavaScript",
+  "homepage": "https://sap.github.io/cloud-sdk/docs/js/overview-cloud-sdk-for-javascript",
+  "license": "Apache-2.0",
+  "private": true,
+  "repository": "github:SAP/cloud-sdk-js",
+  "scripts": {
+    "generate-vdm": "yarn generate-odata-client -i ../../../test-resources/odata-service-specs-e2e/v4 -o .",
```

Comment:
both tests put the code in the same folder - could this lead to conflicts or unexpected behaviour? I would perhaps create two folders with a name indicating canary and v1 and potentially even "memory tes"t in the name.

Reply:
- v1 tests will be generated in `memory-tests/sdk-v1`
- canary tests will be generated in `memory-tests/sdk-canary`

Reply:
sorry I missed the `.` as the location. The reason you do not check it it is that it used canary generator right?

Reply:
Yes, that's the major reason.
Also, we only commit clients for the source code version, so v1 client is not committed.